tensorflow/lite/micro/tools/make/templates/ceva_SP500/ceva_app_makefile.tpl

```diff
@@ -0,0 +1,68 @@
+
+TARGET_TOOLCHAIN_ROOT := /home/yaire/CEVA-ToolBox/V20/SensPro

```

Comment:
Seems like having a hardcoded path is likely an error? I notice this is also present in the other ceva template file as well.